azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/scheduledflowpage.vm

```diff
@@ -129,7 +148,7 @@
               #end
             #else
             <tr>
-              <td colspan="11">No scheduled flow found.</td>
+              <td colspan="12">No scheduled flow found.</td>
```

Comment:
can you elaborate why this is needed? thanks!

Reply:
if colspan="11" and there is no schedule, screenshot is as follows because 1 column(Execution Options) is added

![](https://gyazo.com/8e0abad242ff53a2a73d2f677426883b.png)

Reply:
hah, got it, thanks.